lib/features/email_recovery/presentation/email_recovery_controller.dart

```diff
@@ -338,6 +342,26 @@ class EmailRecoveryController extends BaseController with DateRangePickerMixin {
     popBack();
   }
 
+  Duration getRestorationHorizon() {
+    String restorationHorizon = ((arguments!.session
+        .props[0] as Map<CapabilityIdentifier, CapabilityProperties>?)
+        ?[capabilityDeletedMessagesVault]
+        ?.props[0] as Map<String, dynamic>?)
+        ?['restorationHorizon']
+        ?? "15 days";
+
+    String restorationHorizonValidFormat = "15d";
```

Comment:
currently the server advertizes the horizon like "15 days". i will change it to "15d" because it is what is needed for `parseDuration`

Reply:
i found it made more sense to keep the advertisement that way since we need the human legible string anyway for the content of the banner